src/Discovery/DiscoveryActionBar.tsx

```diff
@@ -483,21 +512,24 @@ const DiscoveryActionBar = (props: Props) => {
           arrowPointAtCenter
           content={(
             <React.Fragment>
-          Directly download data (up to 250Mb) from selected studies
+              {healICPSRLoginNeeded
+                ? 'This dataset is only accessible to users who have authenticated via InCommons. Please log in using the InCommons option.'
```

Comment:
```suggestion
                ? 'This dataset is only accessible to users who have authenticated via InCommon. Please log in using the InCommon option.'
```
Some typos I made in the original ticket, apologies